packages/extension-api/src/extension-api.d.ts

```diff
@@ -1834,6 +1858,70 @@ declare module '@podman-desktop/api' {
     export const clipboard: Clipboard;
   }
 
+  /**
+   * Represents a dictionary object with string keys and values of type T.
+   */
+  export interface Dict<T> {
+    /**
+     * Index signature for accessing values by keys.
+     */
+    [key: string]: T | undefined;
+  }
+
+  /**
+   * Options for running a command.
+   */
+  export interface RunOptions {
+    /**
+     * Environment variables to set for the command.
+     */
+    env?: Dict<string>;
+  }
+
+  /**
+   * Represents the result of running a command.
+   */
+  export interface RunResult {
+    /**
+     * The exit code of the command.
+     */
+    exitCode: number;
+
+    /**
+     * The standard output (stdout) content of the command.
+     */
+    stdOut: string;
+
+    /**
+     * The standard error (stderr) content of the command.
+     */
+    stdErr: string;
+  }
+
+  /**
+   * Namespace for environment-related utilities.
+   */
+  export namespace exec {
```

Comment:
probably rename `exec` into `process`

Reply:
updated the name of namespace, now it called `process`